packages/explorer/src/common.ts

```diff
@@ -9,6 +11,9 @@ export type supportedChainId = supportedChains[supportedChainName]["id"];
 export const chainIdToName = Object.fromEntries(
   Object.entries(supportedChains).map(([chainName, chain]) => [chain.id, chainName]),
 );
+export const supportedChainsIdName = Object.fromEntries(
+  Object.entries(supportedChains).map(([name, chain]) => [chain.id, name]),
+);
```

Comment:
this does the same as `chainIdToName`?

Reply:
yeah, accidental leftover from merge. Will setup knip to detect unused exports